contracts/interface/IIdentity.sol

```diff
@@ -4,4 +4,7 @@ pragma solidity ^0.6.9;
 import "./IERC734.sol";
 import "./IERC735.sol";
 
-interface IIdentity is IERC734, IERC735 {}
+interface IIdentity is IERC734, IERC735 {
+    event ExecutionFailed(uint256 indexed executionId, address indexed to, uint256 indexed value, bytes data);
+    event UpdatedCode(address newContractAddress);
```

Comment:
```suggestion
```

I'm not convinced by this event, but anyway, IIdentity is an interface and is not correlated with our proxy implementation, so UdpdatedCode in IIdentity doesn't make sense.